rest-api-spec/src/main/resources/rest-api-spec/test/search/350_number_field_term.yml

```diff
@@ -0,0 +1,165 @@
+setup:
+    - skip:
+        features: [ "headers" ]
+---
+"search on number fields with doc_values enabled":
+  - do:
+      indices.create:
+        index:  test-iodvq
+        body:
+          mappings:
+            dynamic: false
+            properties:
+              byte:
+                type: byte
+                index: true
+                doc_values: true
```

Comment:
Can you please add a test (can probably just copy/paste this file) that creates these fields with `index: false` to confirm that they are now searchable?

Before this change, that would not have been possible.